content/blog/2018/egress-monitoring-access-control.md

```diff
@@ -0,0 +1,649 @@
+---
+title: Monitoring and Access Policies for HTTP Egress Traffic
+description: Describes how to configure Istio for monitoring and access policies of HTTP egress traffic.
+publishdate: 2018-06-22
+subtitle:
+attribution: Vadim Eisenberg and Ronen Schaffer
+weight: 86
+keywords: [egress, traffic-management, access-control, monitoring]
```

Comment:
No spaces between keywords please.

This may be why Hugo isn't generating a See Also section for this page.